yoti_python_sdk/sandbox/anchor.py

```diff
@@ -0,0 +1,78 @@
+UNKNOWN_EXTENSION = ""
+SOURCE_EXTENSION = "1.3.6.1.4.1.47127.1.1.1"
+VERIFIER_EXTENSION = "1.3.6.1.4.1.47127.1.1.2"
+
+UNKNOWN_ANCHOR_TYPE = "Unknown"
+UNKNOWN_ANCHOR_VALUE = ""
+
```

Comment:
Was going to say that we can just use the constants in `yoti_python_sdk/anchor` instead of these, but it looks like only `UNKNOWN_ANCHOR_TYPE` is being used in this class (unless it's in the integration tests repo?)